### PR TITLE
	Fixed the Bill Page User Search Not Loading in Firefox issue

### PR DESCRIPTION
**Pull request by stevekinyori**
The patient search box now loads when the bill page is loaded
		^cash-198 fixed

### CODE REVIEW

File: omod/src/main/webapp/paymentModes.jsp [hunk 1]
```diff
@@ -14,8 +14,12 @@
   ~ Copyright (C) OpenHMIS.  All Rights Reserved.
   --%>
 <openmrs:require allPrivileges="<%=PrivilegeWebConstants.PAYMENTSMODES_PAGE_PRIVILEDGES%>" otherwise="/login.htm" redirect="/module/openhmis/cashier/paymentModes.form" />
+<spring:htmlEscape defaultHtmlEscape="true" />
+<openmrs:htmlInclude file="/ws/module/openhmis/backboneforms/init.js" />
+<openmrs:htmlInclude file="/moduleResources/openhmis/cashier/js/init.js" />
+<openmrs:htmlInclude file="/moduleResources/openhmis/inventory/js/init.js" />
+<openmrs:htmlInclude file="/moduleResources/openhmis/backboneforms/js/curl.js" />
 <%@ include file="/WEB-INF/template/header.jsp"%>
-<%@ include file="template/localHeader.jsp"%>
```
Comment (akmad) [inline omod/src/main/webapp/paymentModes.jsp]: Why do you include the localHeader content directly in this file?

**Comment by stevekinyori** [comment]
This is because i did not want to include the jquery that we added.. if i do.. you will find that an exception is thrown. 
the main cause is the patient mode fragment that includes its own jquery.js and jquery_ui.

File: omod/web/WEB-INF/web.xml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (akmad) [inline omod/web/WEB-INF/web.xml]: This file (and web/WEB-INF folder) are not needed.

File: omod/src/main/webapp/template/localHeader.jsp [hunk 3]
```diff
@@ -16,3 +16,4 @@
 <openmrs:htmlInclude file="/moduleResources/openhmis/cashier/js/init.js" />
 <openmrs:htmlInclude file="/moduleResources/openhmis/inventory/js/init.js" />
 <openmrs:htmlInclude file="/moduleResources/openhmis/backboneforms/js/curl.js" />
+
```
Comment (akmad) [inline omod/src/main/webapp/template/localHeader.jsp]: Weren't we going to include the jQuery file here?

File: omod/web/WEB-INF/web.xml (hunk 2 above)
Comment (stevekinyori) [inline omod/web/WEB-INF/web.xml]: i get rid of them and recommit?

File: omod/src/main/webapp/template/localHeader.jsp (hunk 3 above)
Comment (stevekinyori) [inline omod/src/main/webapp/template/localHeader.jsp]: actually the jquery has aready been loaded
if you view the page source you will c that we aready have a jquery loaded

**Pull request closed without merging**